src/ansys/dpf/post/simulation.py

```diff
@@ -503,14 +503,17 @@ def _create_dataframe(
                 values = fc.get_available_ids_for_label(label)
                 # Then try to gather the correspond string values for display
                 try:
-                    label_support = self.result_info.qualifier_label_support(label)
-                    names_field = label_support.string_field_support_by_property(
-                        "names"
-                    )
-                    values = [
-                        names_field.get_entity_data_by_id(value)[0] + f" ({value})"
-                        for value in values
-                    ]
+                    if label == "mat":
```

Comment:
I'm not sure this is the right way to do this. Using the code currently in the else block was not working for the "mat" label.

Reply:
@janvonrickenbach here the logic is to expose names whenever available instead of IDs. But I think the logic in the else is dedicated to fluid qualifiers.. I think this is going to need a rework.
I do not think we have access to material names, so what you did is ok for me.
I think in your case the prrevious code was supposed to end-up in the except.